lib/ansible/modules/windows/win_template.py

```diff
@@ -38,8 +38,16 @@
   dest:
     description:
       - Location to render the template to on the remote machine.
-    type: str
+    type: path
```

Comment:
Not sure if this is actually correct

Reply:
It is on-par with Linux and Windows (**win_copy**), so yes, it is correct.